src/OpenTelemetry.Sampler.AWS/Statistics.cs

```diff
@@ -0,0 +1,68 @@
+// <copyright file="Statistics.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Threading;
+
+namespace OpenTelemetry.Sampler.AWS;
+
+internal class Statistics : IDisposable
```

Comment:
For not, I do not see any meaningful usage for this. I suppose part 3 brings this?

Reply:
Yes. The `Statistics` class instance maintains the counter of the requests sampled. I will expand on it's use in the next PR.